### PR TITLE
update(simple-icons): 4.10 update, refinements and export changes

### PR DESCRIPTION
**Pull request by peterblazejewicz**
- export record like object with `get` behaviour, so it can be iterated
  over
- export direct icons via direct module access (*) declaration
- maintainer added
- tests amended
- version bump

https://github.com/simple-icons/simple-icons#usage

Thanks!

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.
- [x] Provide a URL to documentation or source code which provides context for the suggested changes
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## simple-icons ([<kbd>unpkg</kbd>](https://unpkg.com/browse/simple-icons@latest/))
was missing the following properties:
1. .NET
1. 1001Tracklists
1. 1Password
1. 42
1. 4D

<details>
<summary>as well as these 1457 other properties...</summary>
<p>500px, A-Frame, ABB RobotStudio, Ableton Live, About.me, Abstract, Academia, Accusoft, ACM, ActiGraph, Activision, AdBlock, Adblock Plus, AddThis, AdGuard, Adobe, Adobe Acrobat Reader, Adobe After Effects, Adobe Audition, Adobe Creative Cloud, Adobe Dreamweaver, Adobe Fonts, Adobe Illustrator, Adobe InDesign, Adobe Lightroom CC, Adobe Lightroom Classic, Adobe PhoneGap, Adobe Photoshop, Adobe Premiere Pro, Adobe XD, AdonisJS, Aer Lingus, Affinity, Affinity Designer, Affinity Photo, Affinity Publisher, Aiqfome, Air Canada, Air France, Airbnb, Airbus, Aircall, AirPlay Audio, AirPlay Video, Airtable, Alfa Romeo, Algolia, Alipay, AlliedModders, AlloCiné, Alpine Linux, Altium Designer, Amazon, Amazon Alexa, Amazon AWS, Amazon Fire TV, Amazon Lumberyard, Amazon Prime, AMD, American Airlines, American Express, Anaconda, Analogue, Anchor, Andela, Android, Android Auto, Android Studio, AngelList, Angular, Angular Universal, AngularJS, Ansible, Ansys, Ant Design, Antena 3, Apache, Apache Airflow, Apache Ant, Apache Cassandra, Apache CloudStack, Apache Cordova, Apache Druid, Apache ECharts, Apache Flink, Apache Kafka, Apache Maven, Apache NetBeans IDE, Apache OpenOffice, Apache Pulsar, Apache RocketMQ, Apache Solr, Apache Spark, Apollo GraphQL, App Store, Apple, Apple Music, Apple Pay, Apple Podcasts, Apple TV, AppSignal, AppVeyor, ARAL, Arch Linux, Archicad, Archive of Our Own, Ardour, Arduino, Arlo, ArtStation, arXiv, Asana, Asciidoctor, asciinema, Aseprite, ASKfm, ASUS, AT&T, Atari, Atlassian, Atom, Audacity, Audi, Audible, Audio-Technica, Audioboom, Audiomack, Aurelia, Auth0, Authy, Autodesk, Automatic, Autotask, Aventrix, Awesome Lists, awesomeWM, Azure Artifacts, Azure Data Explorer, Azure DevOps, Azure Functions, Azure Pipelines, Babel, Badgr, Badoo, Baidu, Bamboo, Bancontact, Bandcamp, BandLab, Bandsintown, Basecamp, Bata, Bath ASU, Battle.net, BBC iPlayer, Beatport, Beats, Beats by Dre, Behance, Beijing Subway, Bentley, Big Cartel, Bilibili, Bing, Bit, Bitbucket, Bitcoin, Bitdefender, Bitly, Bitrise, Bitwarden, Blackberry, Blender, Blogger, Bloglovin, Bluetooth, BMC Software, BMW, Boeing, Boost, Bootstrap, Bosch, Bose, Bower, Box, Brand.ai, Brandfolder, Brave, Breaker, Broadcom, BT, Buddy, Buefy, Buffer, Bugatti, Bugsnag, Buildkite, Bulma, bunq, Buy Me A Coffee, BuzzFeed, byte, C, C Sharp, C++, Cairo Metro, CakePHP, Campaign Monitor, Canonical, Canva, Car Throttle, Carto, Cash App, Castbox, Castorama, Castro, Caterpillar, CD Projekt, Celery, CentOS, Cesium, CEVO, ChartMogul, Chase, Checkmarx, Chef, Chocolatey, Chupa Chups, Cinema 4D, Circle, CircleCI, Cirrus CI, Cisco, Citrix, Citroën, CiviCRM, Claris, ClickUp, Cliqz, Clockify, Clojure, Cloud 66, CloudBees, CloudCannon, Cloudflare, Cloudsmith, Clubhouse, Clyp, CMake, CNN, Co-op, CocoaPods, Coda, Codacy, Code Climate, Codecademy, CodeChef, Codecov, CodeFactor, Codeforces, CodeIgniter, CodePen, CodersRank, Coderwall, CodeSandbox, Codeship, Codewars, Codio, CoffeeScript, Coinbase, Common Workflow Language, Composer, ComproPago, Concourse, Conda-Forge, Conekta, Confluence, Consul, Contactless Payment, Convertio, Corona Engine, Corona Renderer, Counter-Strike, Coursera, Coveralls, cPanel, Craft CMS, Creative Commons, Crehana, Crowdin, Crunchbase, Crunchyroll, CRYENGINE, CSS Wizardry, CSS3, curl, CurseForge, Cypress, D3.js, DAF, Dailymotion, Daimler, Dark Reader, Dart, Das Erste, Dash, Dashlane, Dassault Systèmes, DataCamp, Datadog, DAZN, dblp, DC Entertainment, Debian, deepin, Deezer, Delicious, Deliveroo, Dell, Deno, Dependabot, Der Spiegel, Designer News, dev.to, DeviantArt, devRant, DHL, Diaspora, Digg, DigitalOcean, Dior, Directus, Discogs, Discord, Discourse, Discover, Disqus, Disroot, Django, DLNA, Docker, DocuSign, Dolby, Douban, Draugiem.lv, Dribbble, Drone, Drooble, Dropbox, Drupal, DS Automobiles, DTube, DuckDuckGo, Dunked, Duolingo, Dynamics 365, Dynatrace, EA, easyJet, eBay, Eclipse IDE, Eclipse Mosquitto, Egnyte, El Jueves, Elastic, Elastic Cloud, Elastic Stack, Elasticsearch, Electron, elementary, Eleventy, Elixir, Ello, Elm, Elsevier, Embarcadero, Ember.js, Emby, Emlakjet, Empire Kred, Envato, EPEL, Epic Games, Epson, Erlang, ESEA, ESLGaming, ESLint, Ethereum, Etsy, Event Store, Eventbrite, Evernote, Everplaces, EVRY, Exercism, Experts Exchange, Expo, EyeEm, F-Droid, F-Secure, Facebook, Facebook Gaming, Facebook Live, FACEIT, Fandango, Fandom, Farfetch, FastAPI, Fastify, Fastly, Favro, FeatHub, Fedora, FedRAMP, Feedly, Ferrari, Ferrari N.V., Fiat, Fido Alliance, FIFA, Figma, figshare, Fila, FileZilla, Firebase, Firefox, Firefox Browser, FIRST, Fitbit, FITE, Fiverr, Flask, Flathub, Flattr, Flickr, Flipboard, Floatplane, Flood, Fluentd, Flutter, Fnac, Font Awesome, Ford, Formstack, Fortinet, Fossa, Fossil SCM, Foursquare, Framer, FreeBSD, freeCodeCamp, freedesktop.org, Freelancer, Fujifilm, Fujitsu, Fur Affinity, Furry Network, G2A, Garmin, Gatling, Gatsby, Gauges, GeeksforGeeks, General Motors, Genius, Gentoo, Geocaching, Gerrit, Ghost, Ghostery, GIMP, GIPHY, Git, Gitea, GitHub, GitHub Actions, GitKraken, GitLab, Gitpod, Gitter, Glassdoor, Glitch, Gmail, GNOME, GNU, GNU Bash, GNU Emacs, GNU IceCat, GNU Privacy Guard, GNU social, Go, Godot Engine, GoFundMe, GOG.com, GoldenLine, Goodreads, Google, Google Ads, Google AdSense, Google Analytics, Google Assistant, Google Calendar, Google Cardboard, Google Cast, Google Chrome, Google Classroom, Google Cloud, Google Drive, Google Earth, Google Fit, Google Hangouts, Google Hangouts Chat, Google Hangouts Meet, Google Keep, Google Lens, Google Maps, Google Messages, Google My Business, Google Nearby, Google News, Google Pay, Google Play, Google Podcasts, Google Scholar, Google Search Console, Google Sheets, Google Street View, Google Tag Manager, Google Translate, GOV.UK, Gradle, Grafana, Graphcool, GraphQL, Grav, Gravatar, GreenSock, Gridsome, Groovy, Groupon, Grunt, Guangzhou Metro, gulp, Gumroad, Gumtree, Gutenberg, Habr, Hack Club, Hackaday, HackerEarth, HackerOne, HackerRank, HackHands, Hackster, HappyCow, Harbor, Hashnode, Haskell, Hatena Bookmark, haveibeenpwned, Haxe, HBO, HCL, HelloFresh, Helly Hansen, Helm, HERE, Heroku, Hexo, HEY, Highly, Hilton, HipChat, Hitachi, Hive, HockeyApp, Home Assistant, HomeAdvisor, Homebrew, homify, Honda, Hootsuite, Hotels.com, Houdini, Houzz, HP, HTML Academy, HTML5, Huawei, HubSpot, Hugo, Hulu, Humble Bundle, Hurriyetemlak, Husqvarna, Hypothesis, Hyundai, Iata, iBeacon, IBM, iCloud, IcoMoon, ICON, Iconfinder, Iconify, IconJar, ICQ, iDEAL, iFixit, iFood, IFTTT, iHeartRadio, IMDb, Imgur, Indeed, InfluxDB, Inkscape, Insomnia, Instacart, Instagram, Instapaper, Instructables, Intel, IntelliJ IDEA, Intercom, Internet Archive, Internet Explorer, InVision, Invoice Ninja, ioBroker, Ionic, iOS, IPFS, Issuu, Itch.io, iTunes, IVECO, Jabber, Jamboard, Jameson, Jasmine, Java, JavaScript, JBL, JCB, Jeep, Jekyll, Jenkins, Jenkins X, Jest, JET, JetBrains, JFrog, JFrog Bintray, Jinja, Jira, Jira Software, John Deere, Joomla, JPEG, jQuery, JR Group, jsDelivr, JSFiddle, JSON, JSON Web Tokens, Julia, Jupyter, Just Eat, JustGiving, Kaggle, Kahoot, KaiOS, Kakao, Karlsruher Verkehrsverbund, Kaspersky, Katana, KDE, KeePassXC, Kentico, Keras, Keybase, KeyCDN, Khan Academy, Khronos Group, Kia, Kibana, Kickstarter, Kik, Kirby, KLM, Klout, Known, Ko-fi, Kodi, Koding, Kofax, Komoot, Kotlin, Krita, KTM, Kubernetes, Kyocera, LabVIEW, Lamborghini, Land Rover, Laravel, Laravel Horizon, Laravel Nova, Last.fm, LastPass, LaTeX, Launchpad, LBRY, Leaflet, Leanpub, LeetCode, Lenovo, Let’s Encrypt, Letterboxd, LG, LGTM, Liberapay, LibraryThing, LibreOffice, libuv, Lighthouse, LINE, LINE WEBTOON, LineageOS, LinkedIn, Linode, Linux, Linux Foundation, Linux Mint, Litecoin, LiveJournal, Livestream, LLVM, LMMS, Logitech, LogMeIn, Logstash, Loom, Loop, Lospec, Lua, Lubuntu, Lufthansa, Lumen, Lyft, MAAS, Macy’s, Magento, Magisk, Mail.Ru, MailChimp, MakerBot, MAN, ManageIQ, Manjaro, Mapbox, MariaDB, MariaDB Foundation, Markdown, Marketo, Marriott, Maserati, MasterCard, Mastodon, Material Design, Material Design Icons, Material-UI, Mathworks, Matrix, Mattermost, Matternet, Maytag, Mazda, McAfee, McDonald's, MDN Web Docs, MediaFire, MediaTemple, Medium, Meetup, MEGA, Mendeley, Mercedes, Messenger, Meteor, Metro de la Ciudad de México, Metro de Madrid, Métro de Paris, micro:bit, Micro.blog, Microgenetics, Microsoft, Microsoft Academic, Microsoft Access, Microsoft Azure, Microsoft Edge, Microsoft Excel, Microsoft Exchange, Microsoft Office, Microsoft OneDrive, Microsoft OneNote, Microsoft Outlook, Microsoft PowerPoint, Microsoft SharePoint, Microsoft SQL Server, Microsoft Teams, Microsoft Visio, Microsoft Word, MicroStrategy, MIDI, Minds, Minecraft, Minetest, Mini, Minutemailer, Mitsubishi, Mix, Mixcloud, Mixer, Mocha, Mojang Studios, Moleculer, Monero, MongoDB, Monkey tie, Monogram, Monster, Monzo, Moo, Moscow Metro, Mozilla, MTA, MTR, MuseScore, MusicBrainz, MX Linux, Myspace, MySQL, N26, NativeScript, NBB, NDR, NEC, Neo4j, Neovim, NestJS, NetApp, Netflix, Netlify, New Relic, New York Times, Next.js, Nextcloud, Nextdoor, NFC, NGINX, niconico, Nim, Nintendo, Nintendo 3DS, Nintendo GameCube, Nintendo Network, Nintendo Switch, Nissan, NixOS, Node-RED, Node.js, Nodemon, Nokia, Notepad++, Notion, Notist, NPM, Nucleo, NuGet, Nuke, NumPy, Nutanix, Nuxt.js, NVIDIA, OBS Studio, OCaml, Octave, Octopus Deploy, Oculus, Odnoklassniki, OnePlus, OnStar, Opel, Open Access, Open Badges, Open Collective, Open Containers Initiative, Open Source Initiative, OpenAI, OpenAI Gym, OpenAPI Initiative, OpenBSD, OpenGL, OpenID, Openlayers, OpenSSL, OpenStack, OpenStreetMap, openSUSE, OpenVPN, Opera, Opsgenie, OpsLevel, Oracle, ORCID, Origin, Osano, Oshkosh, OSMC, Overcast, Overleaf, OVH, Pagekit, PagerDuty, PagSeguro, Palantir, Palo Alto Software, pandas, Pandora, Pantheon, Parity Substrate, Parse.ly, Pastebin, Patreon, Payoneer, PayPal, PeerTube, Pelican, Pepsi, Periscope, Perl, Peugeot, Pexels, Phabricator, Philips Hue, Photocrowd, PHP, Pi-hole, Picarto.TV, PicPay, Pimcore, Pinboard, Pingdom, Pingup, Pinterest, Pivotal Tracker, Piwigo, Pixabay, pixiv, PJSIP, Planet, PlanGrid, Platzi, Player FM, Player.me, PlayStation, PlayStation 2, PlayStation 3, PlayStation 4, PlayStation Vita, Pleroma, Plesk, Plex, Pluralsight, Plurk, Plus Codes, Pocket, Pocket Casts, Pokémon, Poly, Polymer Project, Porsche, PostCSS, PostgreSQL, Postman, Postwoman, Power BI, POWERS, PowerShell, pr.co, pre-commit, PrestaShop, Prettier, Prezi, Prime, Prime Video, Prismic, Probot, ProcessWire, Product Hunt, Prometheus, ProSieben, Proto.io, protocols.io, ProtonMail, Proxmox, Publons, Puppet, PureScript, PyCharm, PyPI, Python, PyTorch, PyUp, Qantas, QEMU, Qgis, Qi, Qiita, QIWI, Qt, Qualcomm, Qualtrics, Quantcast, Quantopian, Quarkus, Quasar, Quest, QuickTime, Quip, Quora, Qwiklabs, Qzone, RabbitMQ, RadioPublic, Rancher, Raspberry Pi, Razer, React, React Router, ReactiveX, ReactOS, Read the Docs, Realm, Reason, Reason Studios, Red Hat, Red Hat Open Shift, Redbubble, Reddit, Redis, Redux, Ren'Py, Renault, RenovateBot, Renren, repl.it, ResearchGate, RetroArch, RetroPie, ReverbNation, Revolut, REWE, Rhinoceros, Ring, Riot, Riot Games, Ripple, Riseup, Roku, rollup.js, Roots, Roots Bedrock, Roots Sage, Roundcube, RSS, RStudio, RTÉ, RTL, RTLZWEI, Ruby, Ruby on Rails, RubyGems, Runkeeper, RunKit, Rust, Ryanair, Safari, Sahibinden, Salesforce, SaltStack, Samsung, Samsung Pay, San Francisco Municipal Railway, SAP, Sass, Sat.1, Sauce Labs, Scala, Scaleway, Scania, scikit-learn, Scopus, Scratch, Scribd, Scrutinizer CI, Seagate, SEAT, Sega, Sellfy, Semantic Web, Semaphore CI, Sencha, Sennheiser, Sensu, Sentry, Server Fault, Serverless, Shanghai Metro, Shazam, Shell, Shenzhen Metro, Shikimori, Shopify, Shopware, Showpad, Shutterstock, Siemens, Signal, Simple Icons, Sina Weibo, SitePoint, Sketch, Sketchfab, SketchUp, Skillshare, ŠKODA, Sky, Skyliner, Skype, Skype for Business, Slack, Slackware, Slashdot, SlickPic, Slides, SlideShare, smart, SmartThings, Smashing Magazine, SMRT, SmugMug, Snapchat, Snapcraft, Snyk, Society6, Socket.io, Sogou, Solus, SonarCloud, SonarLint, SonarQube, SonarSource, Songkick, SonicWall, Sonos, SoundCloud, Source Engine, SourceForge, Sourcegraph, Spacemacs, SpaceX, SparkFun, SparkPost, SPDX, Speaker Deck, Spectrum, Spinnaker, Spinrilla, Splunk, Spotify, Spotlight, Spreaker, Spring, Sprint, SQLite, Square, Square Enix, Squarespace, SSRN, Stack Exchange, Stack Overflow, Stackbit, StackPath, StackShare, Stadia, Staffbase, Starling Bank, Statamic, Staticman, Statuspage, Steam, Steamworks, Steem, Steemit, Steinberg, Stellar, Stencyl, Stitcher, Storify, Storybook, Strapi, Strava, Stripe, strongSwan, StubHub, styled-components, StyleShare, Stylus, Subaru, Sublime Text, Subversion, Super User, Suzuki, Svelte, SVG, SVGO, Swagger, Swarm, Swift, Symantec, Symfony, Symphony, Synology, T-Mobile, Tableau, Tails, Tailwind CSS, Talend, Tapas, Tata, TeamSpeak, TeamViewer, TED, Teespring, TELE5, Telegram, Tencent QQ, Tencent Weibo, TensorFlow, Teradata, Terraform, Tesla, The Conversation, The Mighty, The Models Resource, The Movie Database, The Register, The Sounds Resource, The Spriters Resource, The Washington Post, Threema, Thumbtack, Thunderbird, Tidal, Tide, TikTok, Timescale, Tinder, Todoist, Toggl, Tokyo Metro, Tomorrowland, Topcoder, Toptal, Tor, Tor Browser, Toshiba, Toyota, TrainerRoad, Trakt, Transport for Ireland, Transport for London, Travis CI, Treehouse, Trello, Trend Micro, Tripadvisor, Trove, Trulia, Trusted Shops, Trustpilot, Try It Online, TUI, Tumblr, TuneIn, TurboSquid, Turkish Airlines, Twilio, Twitch, Twitter, Twoo, TypeScript, TYPO3, Uber, Uber Eats, Ubiquiti, Ubisoft, uBlock Origin, Ubuntu, Udacity, Udemy, UIkit, Ulule, Umbraco, Unicode, Unilever, United Airlines, Unity, Unreal Engine, Unsplash, Untangle, Untappd, UpLabs, Uploaded, Upwork, V, V8, Vagrant, Valve, Vauxhall, vBulletin, Veeam, Venmo, Vercel, Veritas, Verizon, vFairs, Viadeo, Viber, Vim, Vimeo, Vine, Virb, VirtualBox, VirusTotal, Visa, Visual Studio, Visual Studio Code, Vivaldi, Vivino, VK, VLC media player, VMware, Vodafone, Volkswagen, Volvo, VSCO, Vue.js, Vuetify, Vulkan, Vultr, W3C, Wagtail, WakaTime, Warner Bros., Wattpad, Waze, Wear OS, Weasyl, WebAssembly, WebAuthn, webcomponents.org, WebGL, Webmin, WebMoney, Webpack, WebRTC, WebStorm, WeChat, WEMO, WhatsApp, When I Work, WhiteSource, Wii, Wii U, Wikimedia Commons, Wikipedia, Windows, Windows 95, Windows XP, Wire, WireGuard, Wish, Wistia, Wix, Wizz Air, Wolfram, Wolfram Language, Wolfram Mathematica, Woo, WooCommerce, WordPress, Workplace, World Health Organization, WP Engine, WP Rocket, write.as, WWE, X-Pack, X.Org, Xamarin, XAML, XAMPP, Xbox, Xcode, XDA Developers, Xero, XFCE, Xiaomi, Xing, XMPP, XRP, XSplit, Y Combinator, Yahoo!, Yamaha Corporation, Yamaha Motor Corporation, Yammer, Yandex, Yarn, Yelp, Yoast, YouTube, YouTube Gaming, YouTube Music, YouTube Studio, YouTube TV, Z-Wave, Zalando, Zapier, ZDF, Zend, Zend Framework, Zendesk, ZeroMQ, Zerply, Zhihu, Zigbee, Zillow, Zingat, Zoom, Zorin, Zulip</p>
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against b488eea0a319c38193d72f3e302fdddda48d46ed
</p>

File: types/simple-icons/simple-icons-tests.ts [hunk 1]
```diff
@@ -1,9 +1,19 @@
-import simpleIcons from 'simple-icons';
+import simpleIcons = require('simple-icons');
+import abbvie = require('simple-icons/icons/abbvie');
```
Comment (ericcornelissen) [inline types/simple-icons/simple-icons-tests.ts:2]: Perhaps it is better to use `simple-icons/icons/simpleicons` here as we will likely never remove it from the collection. This should prevent us from having to update this test if `abbvie` is removed.

**Review by ericcornelissen — CHANGES_REQUESTED** [review]
Thanks for updating this @peterblazejewicz! This looks all correct to me :+1:

I have one minor suggestions but that shouldn't affect much.

**Comment by ericcornelissen** [comment]
Also, I'm not sure how this package is supposed to be versioned but the latest version of simple icons is v4.10.0, for which these type definitions are still valid :thinking:

File: types/simple-icons/simple-icons-tests.ts (hunk 1 above)
Comment (peterblazejewicz) [reply types/simple-icons/simple-icons-tests.ts:2]: Agree, nice suggestion, one sec.

**Comment by peterblazejewicz** [comment]
> Also, I'm not sure how this package is supposed to be versioned but the latest version of simple icons is v4.10.0, for which these type definitions are still valid 🤔

Oh, it's me. Sry. Updated with current released version and also checked project ulr to point to an address that is covered in website cert details (so it works in modern browsers).

Thx!

**Review by ericcornelissen — APPROVED** [review]

**Comment by peterblazejewicz** [comment]
Ready to merge 💋

**Pull request merged**